pkgdown/extra.scss

```
@@ -0,0 +1,236 @@
+  margin-bottom: 10px;
+  padding: 1rem;
+  font-size: 1.5rem;
+  border-top-left-radius: inherit;
+  border-top-right-radius: inherit;
+}
+
+.callout > h3 > .anchor {
+  display: none;
+}
+
+.callout.callout-warning {
+  --callout--border-color: #ffd700;
+  --callout--header-bg: #fffae0;
+  --callout--header-color: #473D00;
+
+  [data-bs-theme="dark"]  & {
+    --callout--border-color: #473D00;
+    --callout--header-color: #fffae0;
+    --callout--header-bg: #473D00;
+  }
+
+  border-color: var(--callout--border-color);
+
+  > h3 {
+    color: var(--callout--header-color);
+    background-color: var(--callout--header-bg);
+
+    &:before {
+      content: "!";
+      font-weight: bold;
+      font-size: 1.25rem;
+      font-family: serif;
+      margin-right: 0.5rem;
+      border-radius: 50%;
+      background-color: #ffd70080;
+      width: 2rem;
+      height: 2rem;
+      display: inline-flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+}
+
+
+.callout.callout-note {
+  --callout--header-color: #373a3c;
+  --callout--header-bg: #e9f2fc;
+  --callout--border-color: #bdccdc;
+
+  [data-bs-theme="dark"]  & {
+    --callout--header-color: #e9f2fc;
+    --callout--header-bg: #1b425d;
+    --callout--border-color: #1b425d;
+  }
+
+  border-color: var(--callout--border-color);
+
+  > h3 {
+    color: var(--callout--header-color);
+    background-color: var(--callout--header-bg);
+
+    &:before {
+      content: "i";
+      font-weight: bold;
+      font-size: 1.25rem;
+      font-style: italic;
+      font-family: serif;
+      margin-right: 0.5rem;
+      border-radius: 50%;
+      background-color: #bdccdc80;
+      width: 2rem;
+      height: 2rem;
+      display: inline-flex;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+}
+
+
+iframe, .bslib-page-fill {
+  height: 500px;
+  border: var(--bs-border-width) solid var(--bs-border-color);
+}
+
+iframe.resizable {
+  resize: vertical;
+  min-height: 200px;
+}
+
+aside .roles,
+footer {
+  color: rgba(var(--bs-emphasis-color-rgb), 0.6);
+}
+
+/* pkgdown fixes for dark mode */
+// these could probably end up in pkgdown?
+
+.card-footer.bg-white {
+  --bs-bg-opacity: 0.1;
+}
+
+.navbar a {
+  text-decoration: none !important;
+}
+
+.navbar-nav .nav-item>.nav-link:hover,
+#toc>.nav a.nav-link:hover,
+#toc>.nav a.nav-link:focus {
+  color: var(--bs-body-color);
+  background-color: rgba(var(--bs-info-rgb), 0.45);
+}
+
+#toc>.nav a.nav-link.active {
+  [data-bs-theme="dark"] {
+    color: var(--bs-body-bg);
+  }
+  background-color: rgba(var(--bs-info-rgb), 0.75);
+}
+
+/* ---- Dark Mode ---- */
+[data-bs-theme="dark"] {
+  .navbar.bg-info {
+    background-color: #333f48 !important;
+  }
+
+  ::selection {
+    background: #344c5a;
+  }
+
+  input[type="search"] {
+    border-color: var(--bs-border-color);
+  }
+
+  pre {
+    background-color: #272b30;
+  }
+
+  // low-tech image softening in dark mode
+  img {
+    opacity: 0.66;
```

I was a bit thrown off by how dim the images looked by default, but that could just be me? 🤷 

```suggestion
    opacity: 0.9;
```

---

Wait until you open it up in the dark on your phone. Which might be a 1% (me) problem. But the idea is that light-mode images can be super glaring when you're looking at an otherwise dark screen. You should be able to hover (or tap on mobile) on a photo to see the full color.

---

Ok yea, that's a good point, maybe I'll get used to it :)